### PR TITLE
Add logging of loggingLevel

### PR DESCRIPTION
**Pull request by Colengms**
Addresses: https://github.com/microsoft/vscode-cpptools/issues/7796

### CODE REVIEW

File: Extension/src/logger.ts [hunk 1]
```diff
@@ -70,6 +71,11 @@ let outputChannel: vscode.OutputChannel | undefined;
 export function getOutputChannel(): vscode.OutputChannel {
     if (!outputChannel) {
         outputChannel = vscode.window.createOutputChannel("C/C++");
+        const settings: CppSettings = new CppSettings();
+        const loggingLevel: string | undefined = settings.loggingLevel;
+        if (!!loggingLevel && loggingLevel !== "None" && loggingLevel !== "Error") {
```
Comment (sean-mcmanus) [inline Extension/src/logger.ts:76]: Can you save the original logging level and re-output it if it's changed to a value that would affect a difference (i.e. switching from None to Error wouldn't matter)?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

**Comment by sean-mcmanus** [comment]
Also, would it be good to add the loggingLevel to the Log Diagnostics logging in cases where the users clear their log or only provide a subset of the C/C++ log?

File: Extension/src/logger.ts (hunk 1 above)
Comment (Colengms) [reply Extension/src/logger.ts:76]: Ok.  I'm not sure adding it also to Log Diagnostics would be useful.  A user could potentially generate the Log Diagnostics from a different session than the output log, making the value unreliable.

Comment (sean-mcmanus) [reply Extension/src/logger.ts:76]: What do you mean by "different session"? I was thinking if the user is in log gathering mode then there's a high probability the Log Diagnostics and C/C++ logging would be close in time, but there is a possibility they change the loggingLevel before or after they do the Log Diagnostics, so it wouldn't be 100% reliable, but could hint at what's probably happening if C/C++ logs appear to be missing info.

**Review by sean-mcmanus — APPROVED** [review]

**Review by michelleangela — APPROVED** [review]

**Pull request merged**